<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MSalman" name="tests.ReactManageGigsTest" tests="6" failures="1" timestamp="20 Mar 2019 09:27:31 GMT" time="59.255" errors="0">
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheDraft_gigs" time="23.511" classname="tests.ReactManageGigsTest"/>
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheDeleted_gigs" time="7.725" classname="tests.ReactManageGigsTest"/>
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_allthePublished_gigs" time="7.690" classname="tests.ReactManageGigsTest"/>
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheClosed_gigs" time="7.743" classname="tests.ReactManageGigsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at pages.ReactManageGigs.verifyClosedGigs(ReactManageGigs.java:154)
	at tests.ReactManageGigsTest.TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheClosed_gigs(ReactManageGigsTest.java:181)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheClosed_gigs -->
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_filter_alltheExpired_gigs" time="7.868" classname="tests.ReactManageGigsTest"/>
  <testcase name="TC_Positive_Verify_thatUser_shouldbe_ableTo_Clear_allthe_gigs" time="4.718" classname="tests.ReactManageGigsTest"/>
</testsuite> <!-- tests.ReactManageGigsTest -->
